File: src/models/body/Vagina.java
package models.body;

import models.items.Item;

/**
 * Vagina, duh
 */
public class Vagina extends Fuckable {
    public Vagina(Item item, String description) {
        super(item, description);
    }

    public Vagina(Item item, String description, int capacity, int sensivity, int wettness) {
        super(item, description, capacity, sensivity, wettness);
    }
}

File: Assets/Scripts/body/Vagina.cs
using items;

namespace body {
	/**
	 * Vagina, duh
	 */
	public class Vagina : Fuckable {
		public Vagina(Item item, string description) : base(item, description) {
	    }

	    public Vagina(Item item, string description, int capacity, int sensivity, int wettness) 
			: base(item, description, capacity, sensivity, wettness) {
	    }
	}
}